File: Allenatore.java
class Allenatore extends Carta{
    private String azione;

    public Allenatore(String a, String nome, String immagine){
        super(nome, immagine, "allenatore");
        this.azione=a;
    }

    public String getAzione(){
        return this.azione;
    }
}

File: PockemonTCGp_Grafica/PockemonTCGp_Grafica/Allenatore.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PockemonTCGp_Grafica
{
    internal class Allenatore : Carta
    {
        private string azione;

        public Allenatore(string a, string nome, string immagine) : base(nome, immagine, "allenatore")
        {
            this.azione = a;
        }

        public string GetAzione()
        {
            return this.azione;
        }
    }
} //---Dzyubanov---
